crates/holochain_sqlite/src/sql/cell/agent_activity/action_ts_to_seq.sql

```diff
@@ -0,0 +1,9 @@
+SELECT
+  seq
+FROM
+  Action
+  JOIN DhtOp ON DhtOp.action_hash = Action.hash
+WHERE
+  DhtOp.authored_timestamp = :authored_timestamp
```

Comment:
This again isn't doing at all what I'm understanding from the provided documentation so far. The logic I'm getting is that we walk the chain backwards until we find a boundary timestamp and then return everything later than it. This actually requires the user to know the timestamp of some action. How do they source that information? It could only come from the action and if you have the action then you know its hash and can use the existing hash bounding functionality.

I'd expect this to look like an `ORDER BY` and `WHERE DhtOp.authored_timestamp > :authored_timestamp`